ci-operator/config/openshift/odo/openshift-odo-master__v4.2.yaml

```diff
@@ -18,6 +18,11 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: odo-secret
+- as: integration-e2e-periodic
+  commands: scripts/openshiftci-presubmit-all-tests.sh
```

Comment:
It might be better to create a new script (for example `openshiftci-periodic-tests.sh`) in odo repo so we can easily control what gets executed from odo repo.
For now, the scripts can be identical.